### PR TITLE
azure-blob-soft-delete

### PR DESCRIPTION
**Pull request by vkagusa-mckinsey**
This PR is test policy to Ensure Soft Delete is Enabled for Azure Storage.

### CODE REVIEW

File: tools/c7n_azure/tests_azure/tests_resources/test_storage.py [hunk 1]
```diff
@@ -650,6 +650,22 @@ def _get_storage_management_client_api_string(self):
...
+        p = self.load_policy({
+            'name': 'test-azure-storage-blob',
+            'resource': 'azure.storage',
+            'filters': [
+                {'type': 'value',
+                'key': 'name',
+                'op': 'glob',
+                'value_type': 'normalize',
+                'value': 'cctstorage*'},
+                {'type': 'soft-delete',
```
Comment (kapilt) [inline tools/c7n_azure/tests_azure/tests_resources/test_storage.py:663]: i think we're missing the filter implementation in the pr.

Comment (vkagusa-mckinsey) [reply tools/c7n_azure/tests_azure/tests_resources/test_storage.py:663]: I added the filter.

**Comment by thisisshi** [comment]
Hi, thanks for the contribution, looks like this is your first contribution. Could you please take a look at and sign the CLA linked above? Direct link here: https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/14202841/52837350/7664/#/?version=2 Thank you!

**Comment by thisisshi** [comment]
@vkagusa-mckinsey looks like it's failing some lint checks

```
flake8 c7n tests tools
tools/c7n_azure/c7n_azure/resources/storage.py:21:43: E231 missing whitespace after ','
tools/c7n_azure/c7n_azure/resources/storage.py:554:1: E302 expected 2 blank lines, found 1
tools/c7n_azure/c7n_azure/resources/storage.py:613:101: E501 line too long (114 > 100 characters)
tools/c7n_azure/c7n_azure/resources/storage.py:616:22: W292 no newline at end of file
tools/c7n_azure/tests_azure/tests_resources/test_storage.py:670:1: E302 expected 2 blank lines, found 1
```

**Comment by thisisshi** [comment]
@vkagusa-mckinsey the lint isuses were on the code that's in the pr a separate pr is not necessary

**Comment by vkagusa-mckinsey** [comment]
> @vkagusa-mckinsey the lint isuses were on the code that's in the pr a separate pr is not necessary

I am not able to make changes in PR that's why I created new PR.

**Comment by thisisshi** [comment]
@vkagusa-mckinsey why aren't you able to make changes in this pr? the 2nd pr you opened doesn't actually address the lint issues in CI

**Comment by vkagusa-mckinsey** [comment]
> @vkagusa-mckinsey why aren't you able to make changes in this pr? the 2nd pr you opened doesn't actually address the lint issues in CI

I made changes in branch. Please can you check now?

**Comment by thisisshi** [comment]
i dont see the changes did you push it?

**Comment by vkagusa-mckinsey** [comment]
these are my changes - https://github.com/vkagusa-mckinsey/cloud-custodian/blob/master/tools/c7n_azure/tests_azure/tests_resources/test_storage.py

**Comment by vkagusa-mckinsey** [comment]
These are my changes - https://github.com/vkagusa-mckinsey/cloud-custodian/blob/azure-blob-soft-delete/tools/c7n_azure/tests_azure/tests_resources/test_storage.py. I am not sure why it's not pulling into PR.

**Comment by vkagusa-mckinsey** [comment]
Please can you check now. I made changes in [vkagusa-mckinsey:master](https://github.com/vkagusa-mckinsey/cloud-custodian)

**Comment by vkagusa-mckinsey** [comment]
I am going to create new PR as having issue in updating this PR.

**Pull request closed without merging**